docs/reference/elasticsearch/rest-apis/retrievers.md

```diff
@@ -44,6 +44,76 @@ The following retrievers are available:
 `rule`
 :   A [retriever](#rule-retriever) that applies contextual [Searching with query rules](/reference/elasticsearch/rest-apis/searching-with-query-rules.md#query-rules) to pin or exclude documents for specific queries.
 
+## Pinned Retriever [pinned-retriever]
+
+A `pinned` retriever returns top documents by always placing specific documents at the top of the results, in the order provided. This is useful for promoting certain documents for particular queries, regardless of their relevance score. The remaining results are filled by a fallback retriever.
+
+#### Parameters [pinned-retriever-parameters]
+
+`ids`
+:   (Optional, array of strings)
+
+    A list of document IDs to pin at the top of the results, in the order provided.
+
+`documents`
+:   (Optional, array of objects)
+
+    A list of objects specifying documents to pin. Each object must contain at least an `_id` field, and may also specify `_index` if pinning documents across multiple indices.
+
+`fallback`
```

Comment:
Apologies, this is another thing I missed in the original review, but `fallback` isn't a great name, I thought we had these as `retriever` like all of our other retrievers. Can we please fix this? Also we should explain that this can be a standard or specialized retriever such as rrf.